<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeMarkEdit</class>
 <widget class="QDialog" name="TimeMarkEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="displayFormat">
        <string>mm:ss.zzz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="framePitch">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Pitch:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="pitchSpin">
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pitchNote">
        <property name="text">
         <string>&lt;note&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>timeEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>TimeMarkEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitchSpin</sender>
   <signal>editingFinished()</signal>
   <receiver>TimeMarkEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
